tzlocal/__init__.py

```diff
@@ -5,13 +5,12 @@
         get_localzone,
         get_localzone_name,
         reload_localzone,
-    )  # pragma: no cover
```

Comment:
OK; I found a syntax for this both black and ruff like, I'll add that back later.

Reply:
We can try that, IIRC coverage didn't handle that correctly. Then again I don't have 100% coverage anyway so I guess it doesn't really matter.